front/webConfig.js

```diff
@@ -0,0 +1,17 @@
+const webConfig = {
+  icons: {
+    defaultColor: "#FFEDE6",
```

Comment:
This must live only in tailwind config.

If you need it here too, then import it from the tailwind config.

Reply:
Removed this key in another PR as it was no longer used